client/main/instance.js

```diff
@@ -181,12 +184,14 @@ const Instance = Backbone.Model.extend({
                             url += `&title=${ encodeURIComponent(options.title) }`;
                     }
                     else if (options.existing) {
-                        url = 'open';
+                        url = 'open?';
```

Comment:
if the `?` here is so you can combine it with the `&lang=` below, i'm not sure `?&` is allowed.